tests/UI/campaigns/functional/BO/09_shipping/01_carriers/02_filterSortAndPagination.js

```diff
@@ -88,7 +88,7 @@ describe('Filter, sort and pagination carriers', async () => {
             testIdentifier: 'filterByName',
             filterType: 'input',
             filterBy: 'name',
-            filterValue: Carriers[1].name,
+            filterValue: Carriers.default.name,
```

Comment:
```suggestion
            filterValue: Carriers.myCarrier.name,
```

Reply:
Default cartier exists too, it's the one that has the same name as the shop